internal/bootstrap/bootstrap_plain_git.go

```diff
@@ -283,22 +282,33 @@ func (b *PlainGitBootstrapper) ReconcileSyncConfig(ctx context.Context, options
 	if _, err = utils.ExecKubectlCommand(ctx, utils.ModeStderrOS, b.kubeconfig, b.kubecontext, kubectlArgs...); err != nil {
 		return err
 	}
-	b.logger.Successf("applied sync manifests")
+	b.logger.Successf("reconciled sync configuration")
+
+	return nil
+}
+
+func (b *PlainGitBootstrapper) ReportKustomizationHealth(ctx context.Context, options sync.Options, pollInterval, timeout time.Duration) error {
+	head, err := b.git.Head()
+	if err != nil {
+		return err
+	}
 
-	// Wait till Kustomization is reconciled
+	b.logger.Waitingf("waiting for Kustomization to be reconciled")
+
+	expectRevision := fmt.Sprintf("%s/%s", options.Branch, head)
 	var k kustomizev1.Kustomization
-	expectRevision := fmt.Sprintf("%s/%s", options.Branch, commit)
 	if err := wait.PollImmediate(pollInterval, timeout, kustomizationReconciled(
 		ctx, b.kube, client.ObjectKey{Name: options.Name, Namespace: options.Namespace}, &k, expectRevision),
 	); err != nil {
-		return fmt.Errorf("failed waiting for Kustomization: %w", err)
+		b.logger.Failuref(err.Error())
+		return err
 	}
 
-	b.logger.Successf("reconciled sync configuration")
+	b.logger.Successf("Kustomization reconciled")
```

Comment:
```suggestion
	b.logger.Successf("Kustomization/%s reconciled", options.Name)
```

Reply:
Added it to the first log message so that it is also logged when things go south.